src/LINEBot/MessageBuilder/Flex/ContainerBuilder/BubbleContainerBuilder.php

```diff
@@ -24,7 +24,7 @@
 use LINE\LINEBot\MessageBuilder\Flex\ContainerBuilder;
 use LINE\LINEBot\MessageBuilder\Flex\BubbleStylesBuilder;
 use LINE\LINEBot\MessageBuilder\Flex\ComponentBuilder\BoxComponentBuilder;
-use LINE\LINEBot\MessageBuilder\Flex\ComponentBuilder\ComponentBuilder;
+use LINE\LINEBot\MessageBuilder\Flex\ComponentBuilder;
```

Comment:
use this at following line.It is not actually used for type checking.
https://github.com/line/line-bot-sdk-php/pull/261/files#diff-f7112712998bcc0715c04639c0ccd0a5R41

same as `LINEBot.php` and other
https://github.com/line/line-bot-sdk-php/blob/5d3a2a7475a1a76f02b5490006b025268e4297b3/src/LINEBot.php